Assets/Plugins/Source/Editor/Utility/UnityProjectUtility.cs

```diff
@@ -101,15 +105,25 @@ private static void FindMissingScripts()
                         // Skip if the component is there.
                         if (component != null) continue;
 
-                        Debug.LogError(
-                            $"Script \"{FullGameObjectPath(go)}\" in scene \"{Path.GetFileNameWithoutExtension(scenePath)}\" is missing.",
-                            go);
+                        missingScriptMessages.Add(($"Script \"{FullGameObjectPath(go)}\" in scene \"{Path.GetFileNameWithoutExtension(scenePath)}\" is missing.", go));
                     }
                 }
             }
 
+            if (0 == missingScriptMessages.Count)
+            {
+                Debug.Log("There were no scripts found to be missing in any scene.");
```

Comment:
I like this log; can we add a similar one for if errors are found? Just something that spits out the message "There were {missingScriptMessages.Count} missing scripts found in scenes".

Reply:
Not certain if you are looking for something other than what is done on lines 119-122. If there are missing scenes, a warning is logged for each one that is missing.

Reply:
I think I know what you mean - like before listing each script that's missing, first just print out how many there are. So I changed the implementation to match that.